app/src/renderer/vuex/modules/user.js

```diff
@@ -16,7 +22,20 @@ export default ({ node }) => {
     address: null,
     errorCollection: false,
     stateLoaded: false, // shows if the persisted state is already loaded. used to prevent overwriting the persisted state before it is loaded
-    error: null
+    error: null,
+
+    // import into state to be able to test easier
+    externals: {
```

Comment:
proposal for all side effects to easily mock them. can also be done on components via data property.